api/routes.py

```diff
@@ -240,21 +241,21 @@ def work():
     annotator = CirculationManagerAnnotator(app.manager.circulation, None)
     return app.manager.urn_lookup.work_lookup(annotator, 'work')
 
-@app.route('/works/contributor/<contributor_name>', defaults=dict(languages=None, audiences=None))
-@app.route('/works/contributor/<contributor_name>/<languages>', defaults=dict(audiences=None))
+@dir_route('/works/contributor/<contributor_name>', defaults=dict(languages=None, audiences=None))
+@dir_route('/works/contributor/<contributor_name>/<languages>', defaults=dict(audiences=None))
```

Comment:
This makes these routes work with or without a slash, which was necessary since the code uses an empty string for audiences when creating the url.